stage1/init/init.go

```diff
@@ -254,6 +260,7 @@ func getArgsEnv(p *stage1commontypes.Pod, flavor string, debug bool, n *networki
 			"noreplace-smp",
 			"systemd.default_standard_error=journal+console",
 			"systemd.default_standard_output=journal+console",
+			"systemd.machine_id=" + machineID,
```

Comment:
Just FYI, it will conflict with https://github.com/coreos/rkt/pull/2684

/cc @jjlakis